docs/manifest.json

```
@@ -240,7 +240,7 @@
               "path": "/docs/api-reference/next.config.js/static-optimization-indicator.md"
             },
             {
-              "title": "Runtime Configuration",
+              "title": "ランタイムの設定",
```

```suggestion
              "title": "ランタイム設定",
```
`runtime config(s)` は「ランタイム設定」とし `runtime configuration` は「ランタイムの設定」として表記を合わせました